sqrl-planner/src/main/java/com/datasqrl/engine/server/VertxEngineFactory.java

```diff
@@ -40,7 +40,7 @@ public static class VertxEngine extends GenericJavaServerEngine {
 
     @Inject
     public VertxEngine(PackageJson packageJson) {
-      super(ENGINE_NAME, packageJson.getEngines().getEngineConfigOrEmpty(ENGINE_NAME));
+      super(ENGINE_NAME, packageJson.getEngines().getEngineConfigOrEmpty(ENGINE_NAME), packageJson);
```

Comment:
Wondering if we should still be performing the `packageJson.getEngines().getEngineConfigOrEmpty` here if we are going to pass the whole `packageJson`


I would say to either just inject `packageJson` OR replace `packageJson` with `packageJson.getEnabledEngines()`

Reply:
My general idea was to pass thorugh `duckdb` and `snowflake` configs from `package.json`, cause that would be the best UX for customers, but for this specific change that seemed too much change, so i'd rather do something like that in the next minor release.

So i added the whole `packageJson` here for future usability, but for now, u're right that like this it's not the best.